requirements.txt

```diff
@@ -2,5 +2,5 @@ git+git://github.com/canonical/operator
 kubernetes
 pyaml
 requests
-lightkube
-lightkube-models
+git+git://github.com/gtsystem/lightkube
```

Comment:
This can just be 

```suggestion
lightkube >= 0.8.1
```

Reply:
Ok. Just waiting to check if integration tests pass. Will change.